client.go

```diff
@@ -60,6 +58,27 @@ var (
 	pathPattern    = regexp.MustCompile(`{[^}]+}`)
 )
 
+// Region specific to Recurly
+type Region string
+
+const (
+    US = Region("us")
+    EU = Region("eu")
+)
+
+// ClientOptions for a new API Client
+type ClientOptions struct {
+    region Region
+}
+
+// APIHost is the base URL to connect on Data Centers for Recurly API v3.
+var (
```

Comment:
Documentation comments in Golang need to be above the variable or function declaration, and match the given case:

```suggestion
var (
    // apiHosts maps the Region to the base URL
```